conn/node.go

```diff
@@ -72,6 +72,9 @@ type Node struct {
 	// The stages are proposed -> committed (accepted by cluster) ->
 	// applied (to PL) -> synced (to BadgerDB).
 	Applied y.WaterMark
+
+	heartbeatsOut int64
```

Comment:
atomic package issues here? This is 64 bit. https://golang.org/pkg/sync/atomic/#pkg-note-BUG